### PR TITLE
Feat(TableV2): add tableV2 part 1

### PR DESCRIPTION
**Pull request by Misael840**
## Summary

Add TableV2 component

## Affected sections

- src/components/TableV2/TableV2.tsx
- src/components/TableV2/base/Header.tsx
- src/components/TableV2/base/HeaderCell.tsx
- src/components/TableV2/base/HeaderRow.tsx
- src/components/TableV2/base/TableBody.tsx
- src/components/TableV2/base/TableCell.tsx
- src/components/TableV2/base/TableRow.tsx
- src/components/TableV2/base/tableV2.css
- src/stories/TableV2.stories.tsx

## How did you test this change?

- Manually

### CODE REVIEW

File: src/components/TableV2/TableV2.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  )
+}
+
+TableV2.Header = Header
+TableV2.HeaderRow = HeaderRow
+TableV2.HeaderCell = HeaderCell
+TableV2.Body = Body
+TableV2.Row = Row
+TableV2.Cell = Cell
+// TableV2.Accordion = Accordion
```
Comment (arielsrodriguez) [inline src/components/TableV2/TableV2.tsx:71]: should this comment go?

File: src/components/TableV2/base/TableBody.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+   *  Body content
+   */
+  children?: React.ReactNode
+}
+
+const Body = (props: BodyProps) => {
+  return (
+    <tbody
+      {...props}
+      className={composeClasses(props.className, 'body text-xs')}
```
Comment (arielsrodriguez) [inline src/components/TableV2/base/TableBody.tsx:15]: Where would the body class come from?

File: src/components/TableV2/base/TableCellText.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      defaultValue={defaultValue}
+      className={composeClasses(
+        'block',
+        props.className,
+        values.className,
+        getTextAlignment()
+      )}
+      onClick={(e) => {
+        e.preventDefault()
+        to && to()
```
Comment (arielsrodriguez) [inline src/components/TableV2/base/TableCellText.tsx:83]:
```suggestion
      to?.()
```

**Review by arielsrodriguez — APPROVED** [review]

File: src/components/TableV2/base/TableBody.tsx (hunk 2 above)
Comment (Misael840) [reply src/components/TableV2/base/TableBody.tsx:15]: It comes from props object

**Pull request merged**